content/en/synthetics/cicd_integrations/github_actions.md

```diff
@@ -0,0 +1,70 @@
+---
+title: GitHub Actions
+kind: documentation
+description: 
```

Comment:
@peculiarism please fill this out!

Reply:
`Integrate Synthetic tests within your GitHub Action workflows using Datadog's Synthetics in CI GitHub Action`